packages/ListBox/stories/Trigger.stories.js

```diff
@@ -1,94 +0,0 @@
-import React from "react";
-import { storiesOf } from "@storybook/react";
-import { getStoryName } from "storybook/storyTree";
-import Button from "@paprika/button";
-import RefreshIcon from "@paprika/icon/lib/Refresh";
-import ListBox from "../src";
-
-const storyName = getStoryName("ListBox");
-const options = ["Punisher", "Catwoman", "Venom", "Thunderbolts", "Deadpool", "Spawn", "Wolverine"];
-
-function renderOptions() {
```

Comment:
Just a note: While usually I'd say this would be more readable to have this function inline, in this specific case where the focus is on the Trigger element, I'd argue this abstraction makes it _easier_ to focus on the important details.